src/Transitioner.js

```diff
@@ -219,9 +245,16 @@ export class Transitioner extends React.Component {
       }
     }
 
+    // Use render container function from last route descriptor
+    const renderContainerFunc = navigationConfig && navigationConfig.navigationOptions
+      && navigationConfig.navigationOptions.renderContainer
```

Comment:
This might be fine, but it seems to neglect the fact that navigationOptions may be a function that returns the options.

Alternatively, you could try using `descriptors[transitionRouteKey].options`, which should have the pre-computed and merged options for the screen.

Reply:
Ok, I'll test this and see if I get the wanted results.

Reply:
Done :-)